apps/angular/bug-effect-signal/README.md

```diff
@@ -0,0 +1,13 @@
+# Bug in Effect ?
+
+> author: thomas-laforge
+
+### Run Application
+
+```bash
+npx nx serve angular-bug-effect-signal
```

Comment:
Maybe `angular-signal-effect-bug` would fit better for the naming, to have `signal` as a prefix for other signal challenges as well, like the pipe challenges, which start with `pipe` as a prefix.

And in the title `Effect in Bug ?` i'd remove the space between `Bug` and `?`. Or simply `Effect Bug` would be nicer imo. I'd avoid question marks in the title. Because until now there is no title with a question mark, which is not really a problem, but it looks cleaner for me.